linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_unlock_time/tests/pam_tally2_absent_account_config.fail.sh

```diff
@@ -7,6 +7,18 @@ account	requisite			pam_deny.so
 account	required			pam_permit.so
 CAPTA
 
+{{% if product in ["sle12","sle15"] %}}
+
+cat >/etc/pam.d/login <<CAPTUTC
```

Comment:
The only change between this jinja2 conditional is the destination file. You can include only the destination file in the conditional and avoid many repeated lines.

Reply:
Actually, the rule is restricted to `prodtype: sle12,sle15`. So, the conditional is not necessary. You can simply update the file.

Reply:
Hello @marcusburghardt 
According to the rule.yml we have only these prodtypes, but why we have multi_platform_ubuntu into bash, ansible and tests ?
Have a nice day
Rumen

Reply:
This probably was missed there when removing `ubuntu` from `prodtype`. Or ubuntu should be included in `prodtype` or the `platform` header should be updated on remediations. @dodys , could you confirm, please?

Reply:
We don't have such rule, the `multi_platform_ubuntu` was added by @teacup-on-rockingchair

Reply:
Thanks @dodys . @rumch-se , in this case you can remove, or even better, update the platform in remediations to `# platform = multi_platform_all`. And remove the conditionals to make it simpler, please.

Reply:
since I started obviously the confusion, I might have added those, since I tested pam_tally2 also on a ubuntu platform and figured it might be useful for older LTS releases, but obviously was wrong, since it remained as a dead code.